romancal/datamodels/rampfitoutput.py

```diff
@@ -0,0 +1,42 @@
+from .core import RomanDataModel
+
+class RampFitOutputModel(RomanDataModel):
+    """
+    A data model for the optional output of the ramp fitting step.
+
+    In the parameter definitions below, `n_int` is the number of
+    integrations, `max_seg` is the maximum number of segments that
```

Comment:
remove `n_int` here and below.

Reply:
Fixed this as well as mentions of nread.